docker/docker-compose.yml

```diff
@@ -1,31 +1,94 @@
-version: "3.7"
+version: "3.8"
 services:
-  minio:
-    image: minio/minio # Stable minio
+  coordinator:
+    image: xain/xain-fl:development
+    build:
+      context: ..
+      dockerfile: docker/Dockerfile
+    depends_on:
+      - minio
+      - redis
+      - influxdb
+    command: /app/coordinator -c /app/config.toml
     volumes:
-      - minio-data:/data
+      - ${PWD}/../configs/docker-dev-coordinator.toml:/app/config.toml
     networks:
       - xain-fl
+    ports:
+      - "5555:5555"
+      - "8081:8081"
+
+  influxdb:
+    image: influxdb:1.8
+    hostname: influxdb
+    container_name: influxdb
     environment:
-      MINIO_ACCESS_KEY: minio
-      MINIO_SECRET_KEY: minio123
+      INFLUXDB_DB: metrics
+      INFLUXDB_DATA_QUERY_LOG_ENABLED: 'false'
+      INFLUXDB_HTTP_LOG_ENABLED: 'false'
+    volumes:
+      - influxdb-data:/var/lib/influxdb
+    networks:
+      - xain-fl
+    expose:
+      - "8086"
+
+  minio:
+    image: minio/minio
+    hostname: minio
+    container_name: minio
+    env_file:
+      - .dev.env
     command: server /data
+    volumes:
+      - minio-data:/data
+    networks:
+      - xain-fl
     ports:
       - "9000:9000"
 
+  setup-initial-buckets:
```

Comment:
I'm not sure if we still need this part. In the current S3 PR, the coordinator is responsible to create the buckets if they don't exist. IMO we can remove this for now and add it back if we need to.